sweet-material-table.html

```diff
@@ -21,11 +21,16 @@
       :host {
         display: block;
       }
+
```

Comment:
I put this directly into sweet-material-table so that sweet-material-table-styles doesn't force the flex on all table columns... But maybe that _should_ be the default behaviour and you have to override it?